extensions/netty/deployment/src/main/java/io/quarkus/netty/deployment/NettyProcessor.java

```diff
@@ -162,8 +162,13 @@ NativeImageConfigBuildItem build(
             log.debug("Not registering Netty native kqueue classes as they were not found");
         }
 
-        builder.addRuntimeReinitializedClass("io.netty.util.internal.PlatformDependent");
-        builder.addRuntimeReinitializedClass("io.netty.util.internal.PlatformDependent0");
+        builder.addRuntimeReinitializedClass("io.netty.util.internal.PlatformDependent")
+                .addRuntimeReinitializedClass("io.netty.util.internal.PlatformDependent0")
+                .addRuntimeReinitializedClass("io.netty.buffer.PooledByteBufAllocator")
+                .addRuntimeReinitializedClass("io.netty.buffer.UnpooledByteBufAllocator")
+                .addRuntimeReinitializedClass("io.netty.buffer.Unpooled")
+                .addRuntimeReinitializedClass("io.vertx.core.http.impl.Http1xServerResponse")
+                .addRuntimeReinitializedClass("io.netty.handler.codec.http.HttpObjectAggregator");
```

Comment:
I am surprised, we use it? In theory Vertx should handle the whole http 1.1 stack  with https://github.com/eclipse-vertx/vert.x/blob/4.x/src/main/java/io/vertx/core/http/impl/HttpServerWorker.java#L284

Reply:
+1 - who's loading this?

Reply:
I will have a look next week (along with the rest of the failing tests).

Reply:
hum it may have found a path that is actually not used. However, adding a substitution might be a bit overkill. So, +1 to re-initialize it.

Reply:
@cescoffier FYI to our so far understanding this happens because when using `--enable-monitoring=jfr` (i.e. `-Dquarkus.native.monitoring=jfr`) GraalVM opens all packages of `java.base` to the unnamed module, which appears to have side-effects on the code reachability of netty.

https://github.com/Karm/mandrel-integration-tests/issues/236#issuecomment-1848206787